src/core/layout/qgslayoutitemlegend.h

```diff
@@ -266,6 +266,26 @@ class CORE_EXPORT QgsLayoutItemLegend : public QgsLayoutItem
      */
     void setSymbolWidth( double width );
 
+    /**
+     * Sets the \a alignment for placement of legend symbols.
+     *
+     * Only Qt::AlignLeft or Qt::AlignRight are supported values.
+     *
+     * \see symbolAlignment()
+     * \since QGIS 3.10.0
```

Comment:
I can't remember us using the minor revision number when it's .0, is that a new trend? :)